rust/main/submitter/src/chain_tx_adapter/transaction.rs

```diff
@@ -0,0 +1,33 @@
+// TODO: re-enable clippy warnings
+#![allow(dead_code)]
+
+use hyperlane_core::H256;
+use uuid::Uuid;
+
+type PayloadId = Uuid;
+type SignerAddress = H256;
+
+/// Full details about a transaction
+pub struct Transaction {
+    uuid: Uuid,                         // unique tx identifier. Used as primary key in the db.
```

Comment:
Rename field from `uuid` to `id` to match `id` field of `PayloadDetails` struct.

```suggestion
    id: Uuid,                         // unique tx identifier. Used as primary key in the db.
```